app/views/shared/fields/_show_raw.erb

```diff
@@ -5,66 +8,71 @@
-<%= render :partial => 'shared/fields/cultural_context' %>
-<%= render :partial => 'shared/fields/function' %>
-<%= render :partial => 'shared/fields/iconography' %>
-<%= render :partial => 'shared/fields/occupation' %>
-<%= render :partial => 'shared/fields/common_name' %>
-<%= render :partial => 'shared/fields/scientific_name' %>
-<%= render :partial => 'shared/fields/style_period' %>
-<%= render :partial => 'shared/fields/technique' %>
-<%= render :partial => 'shared/fields/temporal' %>
+<%= render :partial => 'shared/fields/subject', :locals => component %>
+<%= render :partial => 'shared/fields/built_work_place', :locals => component %>
+<%= render :partial => 'shared/fields/cultural_context', :locals => component %>
+<%= render :partial => 'shared/fields/function', :locals => component %>
+<%= render :partial => 'shared/fields/iconography', :locals => component %>
+<%= render :partial => 'shared/fields/occupation', :locals => component %>
+<%= render :partial => 'shared/fields/common_name', :locals => component %>
+<%= render :partial => 'shared/fields/scientific_name', :locals => component %>
+<%= render :partial => 'shared/fields/style_period', :locals => component %>
+<%= render :partial => 'shared/fields/technique', :locals => component %>
+<%= render :partial => 'shared/fields/temporal', :locals => component %>
 <%= render :partial => 'shared/fields/related_resource',
-        :locals => {:collectionDoc => @document, :docType => "object"} %>
-<%= render :partial => 'shared/fields/custodial_responsibility_note' %>
+        :locals => {:collectionDoc => @document, :docType => 'object'}.merge(component) %>
+<%= render :partial => 'shared/fields/custodial_responsibility_note' , :locals => component%>
 <%= render :partial => 'shared/fields/note',
-        :locals => {:include_label => 'digital object made available by'} %>
-<%= render :partial => 'shared/fields/preferred_citation_note' %>
-<%= render :partial => 'shared/fields/copyright' %>
+        :locals => {:include_label => 'digital object made available by'}.merge(component) %>
+<%= render :partial => 'shared/fields/preferred_citation_note', :locals => component %>
+<%= render :partial => 'shared/fields/copyright', :locals => component %>
 
 <% if can? :update, @document %>
-  <%= render :partial => 'shared/fields/license' %>
-  <%= render :partial => 'shared/fields/statute' %>
-  <%= render :partial => 'shared/fields/other_rights' %>
+  <%= render :partial => 'shared/fields/license', :locals => component %>
+  <%= render :partial => 'shared/fields/statute', :locals => component %>
+  <%= render :partial => 'shared/fields/other_rights', :locals => component %>
 <% end %>
 
-<%= render :partial => 'shared/fields/rights_holder' %>
-<%= render :partial => 'shared/fields/unit' %>
+<%= render :partial => 'shared/fields/rights_holder', :locals => component %>
+<% unless defined?(componentIndex) %>
+  <%= render :partial => 'shared/fields/unit' %>
```

Comment:
for consistency, can the unit partial also use the new Hash#merge approach rather than `unless defined?(componentIndex)`?